README.md

```diff
@@ -8,3 +8,61 @@ It includes Javascript and native stack traces, heap statistics,
 platform information and resource usage etc. With the report enabled,
 reports are triggered on unhandled exceptions, fatal errors, signals
 and calls to a Javascript API.
+
+Usage:
+
+    npm install nodejs/nodereport
+
+    var nodereport = require('nodereport');
+
+By default, this will trigger a NodeReport to be written to the current
```

Comment:
also, I would want to control independently whether nodereport attached to SIGUSR2, or even configure the signal.